### DISCUSSION TITLE
[Feature] Allow multiple selection of faceids to quickly hide face you do not want to manage

### DISCUSSION THREAD

**Discussion by PrayerfulDrop** [immich-app/immich#11200] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be nice to quickly select all of the faces you do not want as part of the "Explore" section.  Current solution requires a select->hide person for each face id.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5142
